torch/quantization/quantization_mappings.py

```diff
@@ -124,6 +124,19 @@ def get_static_quant_module_class(float_module_class, additional_static_quant_ma
         " does not have a corresponding quantized module class"
     return static_quant_module_class
 
+def get_dynamic_quant_module_class(float_module_class, additional_dynamic_quant_mapping=None):
```

Comment:
would be great to add types to function I/O

Reply:
we can add it in a separate PR I think, all other functions in this file are not typed yet

Reply:
it's not blocking this PR, but would be awesome if we started adding these as we go, at least to function I/O.  We don't have to wait for a file to have existing type annots to add more.  This also distributes the cost of adding them to everyone, as opposed to one person.

Reply:
yeah, I'm saying I plan to add it in a separate PR, or are you suggesting to add the type annotations for the functions in this file in this PR?